src/homework/SecondHomeWork.java

```diff
@@ -0,0 +1,65 @@
+package homework;
+
+public class SecondHomeWork {
+
+    public static void main(String[] args) {
+
+        addNumber(10, 20);
+        getRange(-5);
+        getRangeBoolean(-0);
+        printString("Hello", 5);
+        getYear(2012);
+
+
+    }
+
+    public static boolean addNumber(int a, int b) {
+        int sum = a + b;
+        boolean bool;
+        if (sum > 10 && sum <= 20) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
Можно сразу написать
```suggestion
        return (sum > 10 && sum <= 20);
```